src/components/Counter.js

```diff
@@ -6,11 +6,13 @@ class Counter extends Component {
     incrementIfOdd = () => {
         // Stretch Problem: Implement an increment function that
         // only increments if the counter value is odd
+        if (this.props.count % 2 !== 0) {
+            this.props.increment();
```

Comment:
Nice job getting the stretch to work and using modulo to see if the value is odd